src/job/batch_signing.rs

```diff
@@ -112,7 +108,6 @@ pub async fn execute(
             Err(err) => {
                 session.attempt_failed(&err);
                 tracing::error!("{}", err.to_string());
-                last_err = Some(err);
```

Comment:
If a remote signing fails we need to pass the error back up (so that it will be retried). We cannot ignore it entirely. Only if the psbt finalising works can we ignore it.